Directory.Packages.props

```diff
@@ -31,4 +31,4 @@
     <PackageVersion Include="xunit" Version="2.9.1" />
     <PackageVersion Include="xunit.runner.visualstudio" Version="2.8.1" />
   </ItemGroup>
-</Project>
+</Project>
```

Comment:
this one also maybe?

Reply:
I think this one is adding a blank line at the end of the file.